pkg/kubelet/network/plugins.go

```diff
@@ -47,6 +48,20 @@ type NetworkPlugin interface {
 
 	// TearDownPod is the method called before a pod's infra container will be deleted
 	TearDownPod(namespace string, name string, podInfraContainerID kubeletTypes.DockerID) error
+
+	// Status is the method called to obtain the ipv4 and pv6 addresses of the container
```

Comment:
comment on return values, sepcifically that a nil NetworkStatus return means "use docker's info"